frontend/projects/upgrade/src/app/features/dashboard/home/components/modal/import-experiment/import-experiment.component.ts

```diff
@@ -92,31 +110,33 @@ export class ImportExperimentComponent {
       order: 'number',
     }
 
-    let missingProperties = this.checkForMissingProperties({ schema: experimentSchema, data: experiment });
-    if (missingProperties.length > 0) {
-      return false;
-    } else {
-      experiment.conditions.map(condition => {
-        missingProperties = [ ...missingProperties, ...this.checkForMissingProperties({ schema: conditionSchema, data: condition })];
-      });
-
-      if (missingProperties.length > 0) {
-        return false;
-      } else {
-        experiment.partitions.map(partition => {
-          missingProperties = [ ...missingProperties, ...this.checkForMissingProperties({ schema: partitionSchema, data: partition })];
-        });
-        return missingProperties.length === 0;
-      }
+    this.missingProperties = this.checkForMissingProperties({ schema: experimentSchema, data: experiment });
+    let missingPropertiesFlag = true;
+    this.missingAllProperties = this.translate.instant('home.import-experiment.missing-properties.message.text') + this.missingProperties;
```

Comment:
Same as above?

Reply:
removed all other class properties, but `missingAllProperties` is required in html to display errors.